mmv1/products/databasemigrationservice/connectionprofile.yaml

```diff
@@ -78,6 +78,22 @@ examples:
       sqldb_user: 'my-username'
       sqldb_pass: 'my-password'
       profile: 'my-profileid'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'database_migration_service_connection_profile_oracle_forward_ssh'
+    primary_resource_id: 'oracleprofile'
+    ignore_read_extra:
+      - 'oracle.0.password'
+    vars:
+      profile: 'my-profileid'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'database_migration_service_connection_profile_oracle_private'
+    primary_resource_id: 'oracleprofile'
+    ignore_read_extra:
+      - 'oracle.0.password'
+    vars:
+      profile: 'my-profileid'
+      network_name: 'my_network'
```

Comment:
this should have a test_vars_override that overrides this using `acctest.BootstrapSharedTestNetwork`